adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
+from .constants import LETTER_POOL, SCORE_CHART, ALPHABET
 
 def draw_letters():
-    pass
+
+    alphabet_pool = dict(LETTER_POOL)
+    results = []
+
+    while len(results) < 10:
+
+        random_num = randint(0, 25)
+
+        letter = ALPHABET[random_num]
```

Comment:
Notice that the length of `ALPHABET` is 26 letters. This doesn't quite reflect the distribution of letters we expect. 

Your current implementation means I have a 1 in 26 chance of drawing any letter from the pool. However, I should have a higher chance of drawing an `A`, I should have a 9 in 98 chance of drawing an `A` from the pool.

How would you loop over `LETTER_POOL` so that you can create a list of have 98 letters according to the distribution in `LETTER_POOL`? Updating `letters_list` to have 98 letters, instead of 26, will also require that you update the logic related to keeping track of what letters have been drawn.